its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java

```diff
@@ -76,15 +76,18 @@ public void scan_project_sonarqube_scanner_v_3_3() {
 
   @Test
   public void scan_project_ms_build_net_core_scanner() {
-    MSBuildScannerInstallation.install(jenkins, EARLIEST_JENKINS_SUPPORTED_MS_BUILD_VERSION, false);
-    MSBuildScannerInstallation.install(jenkins, MS_BUILD_RECENT_VERSION, true);
+    if (!versionsAreDifferent(EARLIEST_JENKINS_INSTALLABLE_MSBUILD_VERSION, LATEST_JENKINS_MSBUILD_VERSION)) {
+      return;
```

Comment:
This is dangerous: in case the versions are the same, the test will succeed although it won't test anything.

Here you should throw an exception to make sure the test fails and an alarm is raised.

Reply:
You could solve the problem above + this one by replacing your code by:
`checkState(!EARLIEST.equals(LATEST), "Some description of the error");`